src/api/entities/DividendDistribution/index.ts

```diff
@@ -139,6 +147,16 @@ export class DividendDistribution extends CorporateAction {
    */
   public pay: ProcedureMethod<PayDividendsParams, void>;
 
+  /**
+   * Reclaim the Capital Distribution once it is expired
```

Comment:
Reclaim any remaining funds back to the origin Portfolio. This can only be done after the Distribution has expired